hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/utils/FileServiceUtils.java

```diff
@@ -58,28 +64,42 @@ public static void validateContent(@NonNull byte[] content, @NonNull FilesConfig
 
     /**
      * The function validation the file id and returns the file metadata.
```

Comment:
"Validates that the fileId is non-null, not a reserved system Id, and matches a file in the store."

Reply:
we are not doing so as @jsync-swirlds mentioned in multiple PRS is NPM is happening in the first 3 rows there is no matter to apply requiredNonNull.
It also validates if it is a file system 2 lines below.